meson.build

```diff
@@ -155,12 +155,7 @@ orc_dep = dependency('orc-0.4', version: '>= 0.4.17', required: get_option('orcc
 #movidius
 mvncsdk2_dep = dependency('', required: false)
 if not get_option('mvncsdk2-support').disabled()
-  mvncsdk2_dep = cc.find_library('mvnc', required: false)
-  if not mvncsdk2_dep.found()
-    if cc.check_header('mvnc2/mvnc.h', required: get_option('mvncsdk2-support'))
-      mvncsdk2_dep = declare_dependency()
-    endif
-  endif
+  mvncsdk2_dep = dependency('libmvnc', required: get_option('mvncsdk2-support'))
```

Comment:
how about changing  args?
remove this block (line ~155 ~ 159) and change arg for mvnc
```
  'mvncsdk2-support': {
    'extra_deps': [ mvncsdk2_dep ],   <-- 'target': 'libmvnc',
    'project_args': { 'ENABLE_MOVIDIUS_NCSDK2' : 1}
  },
```

Reply:
@jaeyun-jung Sounds good. Updated it.